src/main/java/com/puppycrawl/tools/checkstyle/checks/DescendantTokenCheck.java

```diff
@@ -43,7 +43,8 @@
  * <ul>
  * <li>
  * Property {@code limitedTokens} - Specify set of tokens with limited occurrences as descendants.
- * Type is {@code int[]}.
+ * Type is {@code String[]}.
```

Comment:
For classes in java.lang package we probably can skip fully qualified name, but let's be consistent.
In this PR I see some has fully qualified name for String

Reply:
Done